cohort-engine/src/main/java/com/ibm/cohort/engine/translation/InJVMCqlTranslationProvider.java

```diff
@@ -47,10 +64,17 @@ public InJVMCqlTranslationProvider(LibraryManager libraryManager, ModelManager m
 		this.modelManager = modelManager;
 		this.libraryManager = libraryManager;
 	}
-	
+
 	public InJVMCqlTranslationProvider(LibrarySourceProvider provider) {
+		this(provider, null);
+	}
+	
+	public InJVMCqlTranslationProvider(LibrarySourceProvider provider, File modelInfoFile) {
```

Comment:
I would prefer to have a version of the translate interface that takes the modelInfoFile vs. the constructor. If we end up supporting translation in another provider, it will also need to pass in the modelinfo.

Reply:
Any thoughts on how you would prefer the model info information to be passed along to the `TranslatingLibraryLoader` since that is what is calling `translate()`? I figured changing the `load()` signature wasn't something I should be doing since it comes from the `LibraryLoader` interface from the open source project.

Reply:
Fair point, Thinking more about the problem, it is theoretically possible that we would need more than one custom model info file registered. Perhaps the answer is to put registerModelInfo on the CqlTranslationProvider interface. You would have to remove "static" from the implementation in InJVMCqlTranslationProvider, remove the modelInfo argument from the constructor, and then update the CohortEngine runWithArgs to call the register method on the translationProvider instance directly. If we end up supporting more than one provider that would feel pretty natural.